ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -3616,7 +3616,7 @@ export const testRating = (): Promise<boolean> =>
           },
           {
             emptyIcon: 'album',
-            emptyIconVariant: 'outlined',
+            // emptyIconVariant: 'outlined',
```

Comment:
I'm curious - do we need to keep this around commented out, or can we just delete it?

Reply:
@alex-merkert-olive  Nice catch, this was from other tests but it had an error after compiling process. I commented it out to test the new ocr function and then kinda forgot it's still here 😂   Will look into it and maybe delete it